libcontainer/configs/config.go

```diff
@@ -246,6 +239,109 @@ type Capabilities struct {
 	Ambient []string
 }
 
+//
```

Comment:
Let's move this code block to a separate file (e.g., fsentry.go) inside libcontainer. This way we isolate this code from the rest of the config.

Reply:
Makes sense. Done.